docs/en_US/NAS/HardwareAwareNAS.rst

```diff
@@ -45,3 +46,34 @@ Then, pass this strategy to ``RetiariiExperiment``:
   exp.run(exp_config, port)
 
 In ``exp_config``, ``dummy_input`` is required for tracing shape info.
+
+
+End-to-end ProxylessNAS with latency constraints
+-----------------------------------
+
+`ProxylessNAS <https://arxiv.org/pdf/1812.00332.pdf>`__ is a hardware-aware one-shot NAS algorithm. ProxylessNAS applies the expected latency of the model to build a differentiable metric and design efficient neural network architectures for hardware. The latency loss is added as a regularization term for architecture parameter optimization. In this example, nn-Meter provides a latency estimator to predict expected latency for the mixed operation on other types of mobile and edge hardware. 
+
+To run the one-shot ProxylessNAS demo, first install nn-Meter by running:
```

Comment:
Seems duplicated?

Reply:
Thanks for your comment! I'm wondering if `nn-meter` installation should be repeat twice since there are two different examples. If one user is interested in ProxylessNAS demo, he or she maybe not read the doc of the first example. Another solution is add a new section about `nn-meter` (introduction and installation) at the start of the doc, as both examples applied `nn-meter` package.

Reply:
OK. Then repeat it twice should just work.